### PR TITLE
Document Review Drafts for maintainers

### PR DESCRIPTION
**Pull request by annevk**
See https://github.com/whatwg/whatwg.org/issues/197 for details.

### CODE REVIEW

**Comment by annevk** [comment]
This might need changes depending on the outcome of https://github.com/whatwg/sg/issues/73.

File: MAINTAINERS.md [hunk 1]
```diff
@@ -86,3 +86,12 @@ git merge BRANCH_NAME --ff-only
 This checks out the PR's commits, rebases them on `master`, then fast-forwards `master` to include them.
 
 The `git push --force` line here ensures that the original branch gets updated to sit on top of `master` as well. This ensures GitHub can automatically figure out that the commits were merged, and thus automatically close the pull request with a nice purple "merged" status. So at this point you can do a `git push origin master` to push the changes, and GitHub will close the PR and mark it as merged.
+
+## Review Drafts
+
+As per [Review Drafts](https://whatwg.org/workstream-policy#review-drafts) editors are expected to publish a Review Draft every six months. This is a manual process ([for now](https://github.com/whatwg/sg/issues/74)):
```
Comment (domfarolino) [inline MAINTAINERS.md]: nit: maybe comma after the review drafts link; "As per Review Drafts, ..."?

File: MAINTAINERS.md [hunk 2]
```diff
@@ -86,3 +86,12 @@ git merge BRANCH_NAME --ff-only
...
+
+1. Run `make review` and follow the instructions.
+2. Create a pull request for the new resource and get it reviewed.
```
Comment (domfarolino) [inline MAINTAINERS.md]: nit: maybe make all of these `1.` so they're auto-incremented and no modification to the numbering is necessary if the steps are ever changed?

Comment (annevk) [inline MAINTAINERS.md]: Should encourage a `Meta:` prefix since it doesn't affect the main standard.

File: MAINTAINERS.md [hunk 3]
```diff
@@ -86,3 +86,12 @@ git merge BRANCH_NAME --ff-only
...
+3. Land the pull request and open an issue on the standard's repository to announce the new Review Draft.
```
Comment (domenic) [inline MAINTAINERS.md]: Maybe mention that landing the pull request will automatically publish it to the review-drafts/ URL.

File: MAINTAINERS.md [hunk 4]
```diff
@@ -89,9 +89,10 @@ The `git push --force` line here ensures that the original branch gets updated t
 
 ## Review Drafts
 
-As per [Review Drafts](https://whatwg.org/workstream-policy#review-drafts) editors are expected to publish a Review Draft every six months. This is a manual process ([for now](https://github.com/whatwg/sg/issues/74)):
+As per [Review Drafts](https://whatwg.org/workstream-policy#review-drafts), editors are expected to publish a Review Draft every six months. This is a manual process ([for now](https://github.com/whatwg/sg/issues/74)):
 
 1. Run `make review` and follow the instructions.
```
Comment (foolip) [inline MAINTAINERS.md]: Can you link to an example of `make review`, since it doesn't exist in all Makefiles to begin with?

Comment (annevk) [reply MAINTAINERS.md]: See https://github.com/whatwg/url/pull/387 and https://github.com/whatwg/whatwg.org/pull/201. We should merge those (and equivalent PRs for the former for all others standards) before merging this.

Comment (foolip) [reply MAINTAINERS.md]: OK, made https://github.com/whatwg/fullscreen/pull/125.

**Comment by domenic** [comment]
Some more tweaks, both editorial and based on the discussions we've been having around announcements in https://github.com/whatwg/sg/issues/73.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -16,6 +16,19 @@ Contributions to older issues are always appreciated. If an issue hasn't been up
 
 Leave the **Allow edits from maintainers** option enabled to allow reviewers to fix trivial issues directly on your branch rather than needing to write review comments asking you make the edits. For more details, see [Improving collaboration with forks](https://github.com/blog/2247-improving-collaboration-with-forks) in the GitHub Blog.
 
+## Building
+
+All WHATWG standards, apart from HTML, have a `Makefile` in their repository to help with building them. You'll need `make` and [`curl`](https://curl.haxx.se/) installed at the minimum; ideally you'll also have a full Bash shell. These prerequisites should be present on most Linux and macOS systems already. For Windows, we recommend [Git Bash](https://gitforwindows.org/) and [Make for Windows](http://gnuwin32.sourceforge.net/packages/make.htm).
+
+Once you have the prerequisites, you can run `make` in the standard's directory to build the spec; it will appear as a `.html` file in the same directory. This will call out to a web service to build the standard, so be sure you're connected to the internet.
```
[the remark this answers is not in the document]
Comment (domenic) [reply CONTRIBUTING.md]: Right, the plans are to change this, see https://github.com/whatwg/url/pull/387

**Pull request merged**